Slim/Formats/Playlists/CUE.pm

```diff
@@ -337,9 +860,118 @@ sub parse {
 		return;
 	}
 
+	my $msg = {};
+	$msg->{'message'}	= 'end of parse';
+	$msg->{'tracks'}		= $tracks;
+	main::DEBUGLOG && $log->is_debug && $log->debug(Data::Dump::dump($msg));return $tracks;
+
 	return $tracks;
 }
 
+sub _addCommand{
+	my $cuesheet	= shift;
+	my $tracks		= shift;
+	my $inAlbum		= shift;
+	my $currtrack	= shift;
+	my $command		= shift;
+	my $value		= shift;
+
+	if ($inAlbum and !defined $cuesheet->{$command}){
```

Comment:
Really? I was thinking both versions are short-circuiting, according to docs, but ok, I'll do.